<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoKnowledgeAIDockWidgetBase</class>
 <widget class="QDockWidget" name="GeoKnowledgeAIDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Geo Knowledge AI</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>84</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Chatbot:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widgetChatbotParent" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>My Question:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextEditorInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnSendOrTerminate">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="btnSetting">
      <property name="toolTip">
       <string>Settings</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/plugins/geo_knowledge_ai/image/settings.svg</normaloff>:/plugins/geo_knowledge_ai/image/settings.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btnHistory">
      <property name="toolTip">
       <string>History List</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/plugins/geo_knowledge_ai/image/list.svg</normaloff>:/plugins/geo_knowledge_ai/image/list.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
